src/main/java/gui/DrawableNode.java

```diff
@@ -0,0 +1,65 @@
+package gui;
+
+import com.sun.corba.se.impl.orbutil.graph.Graph;
+import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.paint.Color;
+
+/**
+ * Created by TUDelft SID on 17-5-2017.
+ */
+public class DrawableNode {
```

Comment:
Tested this class partly, found some trouble testing with the GraphicContext class.